src/app/cheat-sheets/game-base/space-age/space-age.component.html

```diff
@@ -63,7 +63,12 @@ <h4 class="card-title" id="space-platforms">
         </p>
       </li>
       <li>
-        While you cannot bring anything in your inventory, you can definitely load up the space platform with starter items for next planet
+        While you cannot bring anything in your inventory directly, you can load up space platforms with items which can be dropped on your planet of choice - you do not need a landing pad.
+        <ul>
+          <li>
+            Additionally, you can also craft items in space like you can on the ground - for instance, while you cannot launch a rocket silo inside a rocket, you can launch the required resources for one.
```

Comment:
```suggestion
            You can craft items in space, like you can on the ground. For example: the rocket silo is too heavy for your rocket. But you can launch the items needed to build the rocket silo, and then craft the silo in space.
```

Can you handcraft items when on the Space platform?

Or should we recommend users launch the rocket silo items, then drop them on the planet, and craft the silo on the _planet_?

Reply:
Rocket silo has a weight of 10 tons. (cannot be launched) I meant crafting as in using an assembling machine. You can't handcraft items there.